tests/queries/1_stateful/00056_view.reference

```diff
@@ -19,13 +19,13 @@
 59183	85379
 33010362	77807
 800784	77492
-1704509	523264
```

Comment:
But note, that there was one test for existing behavior, so maybe old behavior was desired...